### PR TITLE
ESA-1572 Unit Test

### PR DESCRIPTION
**Pull request by afreen1701**

### CODE REVIEW

File: force-app/main/default/classes/FaxServiceTest.cls [hunk 1]
```diff
@@ -1,10 +1,329 @@
...
+static void test_Send_Fax_pass() {
+    EmailTemplate template = new EmailTemplate();
+    template.developername  = 'Approved_Fax_Template1'; 
+    template.name = 'Approved Fax';
+    template.templatetype = 'custom';
+    template.FolderId= UserInfo.getUserId();
+    insert template;
+
+    Case record = [SELECT Id,status,Fax_Sent_Date__c FROM Case WHERE status=:'Approved' LIMIT 1];
+    EmailTemplate temp = [SELECT Id,name ,developername from EmailTemplate where developername =:'Approved_Fax_Template1'];
```
Comment (janjulian) [inline force-app/main/default/classes/FaxServiceTest.cls:26]: There are no email templates with a developername of `Approved_Fax_Template1`. Please use any of the email templates located in `\MoH-SAT\force-app\main\default\email\SA_Fax_Templates\`

File: force-app/main/default/classes/FaxServiceTest.cls [hunk 2]
```diff
@@ -1,10 +1,329 @@
...
-        
+  @TestSetup
+  static void makeData(){
+    Case record = new Case();
+    record.status='Approved';
+    record.Provider_Fax__c='8558120171';
+    insert record; 
+
+    Case record2 = new Case();
+    record2.status='Approved2';
```
Comment (janjulian) [inline force-app/main/default/classes/FaxServiceTest.cls:11]: You have to use a valid status value, which `Approved2` isn't.

**Review by janjulian — CHANGES_REQUESTED** [review]
Please review comments and fix on next commit;

File: force-app/main/default/classes/FaxServiceTest.cls [hunk 3]
```diff
@@ -1,10 +1,329 @@
...
+    EmailTemplate temp = [SELECT Id,name ,developername from EmailTemplate where developername =:'Approved_Fax_Template1'];
+    Id templateId = temp.id;
+    Id caseId=records.id;
+
+    String result =  FaxService.generatePdf(caseId,templateId);
+    System.assert(result!=null);
+}
+
+  @isTest
+static void test_buildPaylload() {
```
Comment (youngbeanie) [inline force-app/main/default/classes/FaxServiceTest.cls:119]: May be a typo in method name.

**Review by janjulian — APPROVED** [review]
Please merge if build check is green

**Pull request merged**